innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+* InnerSource Portal
+
+## Patlet
+
+InnerSource projects at your company are increasing but potential contributors have no easy way of discovering them.
+
+Create an intranet website that indexes all available InnerSource projects.  Make it easy for InnerSource projects owners to include information about their projects and for visitors to the site to search across all the available projects.  This will enable potential contributors to more easily locate projects of interests and learn more about who is running them, the project goals and access technical and community management related documentation.
+
+## Problem
+
+There is no central, easily accessible resource for InnerSource project owners and potential contributors to leverage to advertise or discover InnerSource projects.  
+
+## Story
+
+You are attempting to establish an InnerSource practice within your organization.  You are aware of some projects being run using an InnerSource model, but their existence is only being communicated via word of mouth, email or sidebar conversations with other employees.  There is no single, shared resource for employees across the organization to access which will allow them to easily discover all ongoing InnerSource projects.  This is severely limiting the growth potential of every InnerSource project.  What can be done to help all InnerSource projects raise their visibility to as large an audience as possible and attract contributors organization-wide?
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+
+## Context
```

Comment:
Tools like GitHub and Bitbucket offer limited support for discovering projects, if the overall number of projects is very high. GitHub offers a few marketing tools, but nothing like what you propose. I feel that should go in the context as well.